packages/adp-tooling/test/unit/preview/adp-preview.test.ts

```diff
@@ -197,15 +197,14 @@ describe('AdaptationProject', () => {
         });
 
         test('/local.file', async () => {
-            const testFileContent = '~test';
+            const localPath = '/local.file';
```

Comment:
This is not really testing the string replace. Maybe one additional test?

Reply:
The test mocks the `byGlob` so I don't think two tests improve the quality, however, I have modified the one to cover the different file extensions scenario with https://github.com/SAP/open-ux-tools/pull/1686/commits/d40c3112ecbde5120f47f7fd336150338624a71b

Reply:
Thanks!